src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4902,6 +4903,22 @@ void ReportZoneMeanAirTemp(EnergyPlusData &state)
     // PURPOSE OF THIS SUBROUTINE:
     // This subroutine updates the report variables for the AirHeatBalance.
 
+    for (int ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) {
```

Comment:
This is good, but it needs to be protected by a oneTime variable so these loops don't execute every timestep.

Reply:
Oh make sense. I will add the oneTime flag.

Reply:
added